cloverage/src/cloverage/coverage.clj

```diff
@@ -131,10 +135,14 @@
         add-test-nses (:extra-test-ns opts)
         ns-regexs     (map re-pattern (:ns-regexp opts))
         test-regexs   (map re-pattern (:test-ns-regexp opts))
+        exclude-regex (map re-pattern (:ns-exclude-regex opts))
         start         (System/currentTimeMillis)
         test-nses     (concat add-test-nses (find-nses test-regexs))
-        namespaces    (concat add-nses      (find-nses ns-regexs))
-        ]
+        namespaces    (clojure.set/difference
```

Comment:
Given clojure.set is explicitly imported, can this please become 'set' not 'clojure.set'?